client/my-sites/promote-post-i2/main.tsx

```diff
@@ -126,12 +133,20 @@ export default function PromotedPosts( { tab }: Props ) {
 		{
 			id: 'posts',
 			name: translate( 'Ready to promote' ),
-			itemCount: totalPostsUnfiltered || null,
+			itemCount: totalPostsUnfiltered,
 		},
 		{
 			id: 'campaigns',
 			name: translate( 'Campaigns' ),
-			itemCount: totalCampaignsUnfiltered || null,
+			itemCount: totalCampaignsUnfiltered,
+		},
+		{
+			id: 'credits',
+			name: translate( 'Credits' ),
+			className: 'pull-right',
+			itemCount: creditBalance,
+			isCountAmount: true,
+			enabled: creditBalance !== undefined && creditBalance !== 0,
```

Comment:
Just thinking out loud. It's interesting whether it will be even more right to use `creditBalance > 0` instead of `creditBalance !== 0`. Assuming that dealing with floating point would work better with `>` operator. But the current solution looks workable either.

Reply:
You are right; we won't be having negative credits, so its safe to make the change to `>` instead of the equals. 
I need to maintain the `creditBalance !== undefined` to differentiate the case between an invalid amount (NaN) and a blank response from the API. 

If by any reason the backend start sending invalid balances, the interface will display the tab but without an amount, and that will give us a clue where the error could be.

Reply:
I change my mind regarding showing the page if we aren't able to parse the balance that the backend gives us. 

I will go for your solution and only show the tab/section if we have a valid amount.